CONTRIBUTING.md

```diff
@@ -0,0 +1,49 @@
+# Contributing
+
+Thanks for considering contributing to leomvit.github.io!
+
+## Opening issues
+
+If you find a bug, please feel free to [open an issue](https://github.com/leomvit/leomvit.github.io/issues).
+
+If you taking the time to mention a problem, even a seemingly minor one, it is greatly appreciated, and a totally valid contribution to this project. Thank you!
+
+## Fixing bugs
+
+We love pull requests. Here’s a quick guide:
+
+1. [Fork this repository](https://github.com/leomvit/leomvit.github.io/fork) and then clone it locally:
+
+  ```bash
+  git clone https://github.com/abcd/leomvit.github.io
+  ```
+
+2. Create a topic branch for your changes:
+
+  ```bash
+  git checkout -b fix-for-that-thing
+  ```
+
+3. Commit a fix that makes the test pass:
+
+  ```bash
+  git commit -am "Adds a fix for that thing!"
+  ```
+
+4. Run the tests, and check if the website looks good
+
+5. If everything looks good, push to your fork:
+
+  ```bash
+  git push origin fix-for-that-thing
+  ```
+
+6. [Submit a pull request.](https://help.github.com/articles/creating-a-pull-request)
+
+7. Enjoy being the wonderful person you are
+
+## Adding new features
+
+Thinking of adding a new feature? Cool! [Open an issue](https://github.com/leomvit/leomvit.github.io/issues) and let’s design it together.
+
+**Thanks to [default](https://github.com/kennethormandy/default) for the template!**
```

Comment:
Hi @neelr . Thanks for adding this file! @AvikantSrivastava just added a pull request template and an issue template to this project (see #93 ). Can you mention those somewhere here, instead of the default template? Also I am requesting an additional review from @AvikantSrivastava , he may just want you to add specific details relevant to the project or the community if necessary.